pkg/manager/http.go

```diff
@@ -734,6 +738,42 @@ func (serv *HTTPServer) modulesInfo(w http.ResponseWriter, r *http.Request) {
 	serv.jsonPage(w, r, "modules", cover.Modules)
 }
 
+func (serv *HTTPServer) httpAddInput(w http.ResponseWriter, r *http.Request) {
+	w.Header().Set("Content-Type", "multipart/form-data")
```

Comment:
Why did you need to set the Content-Type header? The response generated by this method does not seem to contain any multipart data.

Reply:
right , client should do ,removed.